docs/source/configuration/settings-reference.md

```diff
@@ -206,7 +206,13 @@ controlpanels
     The group can be one of the default groups 'General', 'Content', 'Security', 'Add-on Configuration', 'Users and Groups' or a custom group.
 
 filterControlPanelsSchema
-    A schema factory for a control panel. It is used internally, to tweak the schemas provided by the controlpanel endpoint, to make them fit for Volto.
+    A schema factory for a control panel.
+    It is used internally, to tweak the schemas provided by the `@controlpanels` endpoint, making them fit for Volto.
+    It uses the `unwantedControlPanelsFields` setting.
+
+unwantedControlPanelsFields
+    Control panels fields that are not used in Volto.
```

Comment:
```suggestion
    Control panel fields that are not used in Volto.
```

Reply:
The fields can be from several control panels. For example: navigation, site...

That's why it's in the plural. @davisagli  what is your opinion here?

Reply:
Ah, in that case, it should be something else, using the plural possessive form. I edited my suggestion.